rero_ils/modules/loans/api.py

```diff
@@ -665,19 +665,35 @@ def get_due_soon_loans():
     return due_soon_loans
 
 
-def get_overdue_loans():
-    """Return all overdue loans."""
-    overdue_loans = []
-    results = current_circulation.loan_search_cls()\
-        .filter('term', state=LoanState.ITEM_ON_LOAN)\
-        .params(preserve_order=True)\
-        .sort({'_created': {'order': 'asc'}})\
+def get_overdue_loan_pids(patron_pid=None):
+    """Return all overdue loan pids optionally filtered for a patron pid.
+
+    :param patron_pid: the patron pid. If none, return all overdue loans.
```

Comment:
the docs does not talk about the `range` filter

Reply:
the docstring talks about "overdue loan". An overdue_loan is a loan with ITEM_ON_LOAN state with a enddate <= systdate. This is the purpose of the 'range' filter. This filter is not a function argument.